src/endpoints/providers/entities/provider.query.options.ts

```diff
@@ -0,0 +1,18 @@
+import { BadRequestException } from "@nestjs/common";
+
+export class ProviderQueryOptions {
+  constructor(init?: Partial<ProviderQueryOptions>) {
+    Object.assign(this, init);
+  }
+
+  withLatestInfo?: boolean = false;
+  withIdentityInfo?: boolean = false;
+
+  static applyDefaultOptions(owner: string | undefined, options: ProviderQueryOptions): ProviderQueryOptions {
+    if (options.withLatestInfo === true && owner === undefined) {
```

Comment:
`&& !owner` to also take into account situations where it is string empty and it may bypass this check